index.html

```diff
@@ -1,20 +1,45 @@
-<p><strong>Here is how I decided what I wanted to be in life--An Entrepeneur:</strong></p>
-<p>When I learned more about the shoe community, I saw lots of business opportunities it that area. As I learned more about shoes, I would constantly see the word entrepreneur/enthusiast tied in with other sneakerheads' bios and such.</p>
-<h1><strong><center>I LOVE SNEAKERS!</center></strong></h1>
-<address>
-	Name: Nicole Sprague<br>
-	Email: spragueskicks@yahoo.com<br>
-	Number: (217) 552-0587
-</address>
-<p>
-	<img src="http://www.flightclub.com/" alt="Smiley face" Style="float:left;width:42px;height:42px;">
-</p>
\ No newline at end of file
+    <head>
+    </head>
+    <body style="background-color:maroon;">
+        <h1 style="color:white;">
+            <center><strong>Nicole Sprague</strong></center>
+        </h1>
+        <h2 style="font-family:courier;">
+            <center>The 15 year old sneakerhead/entrepreneur.</center>
+        </h2>
+        <a href="http://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwi4nceApLzJAhWHOIgKHYkFCnMQjRwIBw&url=http%3A%2F%2Fwww.highsnobiety.com%2F2013%2F03%2F26%2Fair-jordan-4-toro-bravo%2F&psig=AFQjCNGJyMIZyp1SiAXWnrbbwAtCQjLujg&ust=1449113698064619">Here is a picture of my favorite shoes!</a>
+        <p>
+            <strong>Here is a little history on how I got into shoes:</strong>
+        </p>
+        <p>
+            When I was just a few weeks old, I started going to a daycare in my local town.
+            I basically lived there for my whole life before I got into school.
+            While there, I met my best friend, Rachel, now being 15 years old we are still best friends.
+            One day as we were texting, she sent me a picture of her new Jordan's. They were the Son of Mars Low.
+            I instantly fell in love with them, and decided to do some more research on shoes.
+            Ever since then, I've been obsessed with shoes, I now own over 10 pairs of kicks in my collection.
+            I hope to expand that collection in the future.
+        </p>
+        <p>
+            <strong>Here is how I decided what I wanted to be in life--An Entrepeneur:</strong>
+        </p>
+        <p>
+            When I learned more about the shoe community, I saw lots of business opportunities it that area.
+            As I learned more about shoes, I would constantly see the word entrepreneur/enthusiast tied in with
+            other sneakerheads' bios and such.
+        </p>
+        <h1>
+            <strong><center>I LOVE SNEAKERS!</center></strong>
+        </h1>
+        <address>
+        	Name: Nicole Sprague<br>
+        	Email: spragueskicks@yahoo.com<br>
+        	Number: (217) 552-0587
+        </address>
+        <p>
+        	<img src="http://www.flightclub.com/" alt="Smiley face" Style="float:left;width:42px;height:42px;" />
```

Comment:
Image tags are a little quirky - they are self-closing.

example _note the forward slash - just before the closing bracket_::
`<img src="awesme-pic-of-pizza.png" alt="this pic makes me hungry!"  />`